### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by pascalberger**
Clean up code and fix code analysis issues

### CODE REVIEW

File: src/BBT.StructureTools/Copy/ICopyChildInterception.cs [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
     public interface ICopyChildInterception<in TClassToCopy> : IBaseAdditionalProcessing
         where TClassToCopy : class
     {
         /// <summary>
-        /// This method will called before the copy process of the model in the type parameter starts.
+        /// This method will be called before the copy process of the model in the type parameter starts.
         /// </summary>
         /// <returns><code>True</code> if the model must not copy, otherwise. <code>False</code></returns>
-        bool CalcopyChild(TClassToCopy source, TClassToCopy target);
+        bool BeforeCopyingChild(TClassToCopy source, TClassToCopy target);
```
Comment (pascalberger) [inline src/BBT.StructureTools/Copy/ICopyChildInterception.cs]: Not sure if this is the correct name here

Comment (marco-bertschi) [reply src/BBT.StructureTools/Copy/ICopyChildInterception.cs]: `ShallCopyChild` or something would probably be a better name.

Comment (marco-bertschi) [reply src/BBT.StructureTools/Copy/ICopyChildInterception.cs]: To add a bit of context, Interceptions are a way of controlling the copy process dynamically based on the actual data, and either continue or stop at a given "branch" of the tree.

**Review by marco-bertschi — COMMENTED** [review]
Had a quick look, meeting now. I'll check this afternoon again.

**Review by marco-bertschi — APPROVED** [review]

**Pull request merged**